framework/Runners/DistributedMemoryRunner.py

```diff
@@ -26,6 +26,14 @@
 import copy
 import sys
 import abc
+from importlib import util as imutil
+## TODO: REMOVE WHEN RAY AVAILABLE FOR WINDOWOS
+_rayAvail = False if imutil.find_spec("ray") is None else True
```

Comment:
Could could be abstracted to a utility method for an arbitrary library presence check? That seems like it would be useful.